### PR TITLE
Importing Selenium IDE recordings side project into Golem project #216

### PR DESCRIPTION
**Pull request by ATTO-RATHORE**
Add feature for Converting/Importing Selenium IDE recording side project into Golem Project, Suite, Test script.

### CODE REVIEW

File: golem/gui/static/js/index.js [hunk 1]
```diff
@@ -43,4 +54,43 @@ function createProject() {
...
+        $.ajax({
+            type: 'POST',
+            url: '/api/uploadside',
+            data: form_data,
+            contentType: false,
+            cache: false,
+            processData: false,
+            async: false,
+            success: function(data) {	
+				if(data.errors.length == 0) {
```
Comment (luciano-renzi) [inline golem/gui/static/js/index.js:82]: the code here should be indented properly

**Review by luciano-renzi — COMMENTED** [review]
Thank you for your submission. I've marked a few outstanding things in the code. But also:
* the code formatting should respect pep8 (https://www.python.org/dev/peps/pep-0008/) if you are using an IDE like Pycharm or Visual Studio Code these problems should be outlined for you and at least in Pycharm there's an option to 'reformat code'
* there are no unit tests. There should be a minimum of unit test coverage for new code

File: golem/core/suite.py [hunk 2]
```diff
@@ -8,12 +8,11 @@
 from golem.core.project import Project, validate_project_element_name, BaseProjectElement
 
 
-def create_suite(project_name, suite_name):
-    suite_content = ('\n'
+def create_suite(project_name, suite_name,suite_content = ('\n'
```
[the remark this answers is not in the document]
Comment (Sunil-Rathore) [reply golem/core/suite.py:11]: sometime suite need to create for new project import through selenium side file. so in this case need to pass suite_content. through edit_suite i cant create suite.

File: golem/core/test.py [hunk 3]
```diff
@@ -8,12 +8,11 @@
 from golem.core import parsing_utils
 
 
-def create_test(project_name, test_name):
```
[the remark this answers is not in the document]
Comment (Sunil-Rathore) [reply golem/core/test.py:11]: same as create_test also required to create new test if imported new project through selenium ide file. and it's required the test_content to pass for creating the test.

**Comment by luciano-renzi** [comment]
Stale, closing. Thank you.

**Pull request closed without merging**